### DISCUSSION TITLE
How come I can't replicate the comfy UI picture even with the same setting and seed?

### DISCUSSION THREAD

**Discussion by weirdlighthouse** [vladmandic/sdnext#1963] [Q&A] [answered]
![ComfyUI_00010_](https://github.com/vladmandic/automatic/assets/140368252/14850389-843f-435f-b551-5194ed47ea8f)

Vlad diffusion picture
![00177-a visually stunning composition featuring the Joker seated](https://github.com/vladmandic/automatic/assets/140368252/5821f0e6-7acb-405c-b081-175025f341b7)

**Comment by soyatlas** [comment]
Hello,
Sorry my bad english, I will try explain you it.

In WebUI **settings** go to "**Stable Diffusion**" menu, at last you see this:
"Random number generator source. (changes seeds drastically; use CPU to produce the same picture across different videocard vendors)"
Down to this text select CPU not the GPU
This is a trick that I had read in COMFYUI comunity.

Hope it help you.

**Reply by maforget** [reply]
This works with the AUTO1111 version, but not this fork. Option seems to have been removed.

**Reply by xedis** [reply]
@soyatlas  and @weirdlighthouse try this button:
![image](https://github.com/vladmandic/automatic/assets/4144634/832dbb06-2ea7-47dc-aef2-4a8b863299f0)


![03865-a visually stunning composition featuring the Joker seated](https://github.com/vladmandic/automatic/assets/4144634/e9635b6b-37c4-4086-b2eb-92f84fd2fabc)
![03864-a visually stunning composition featuring the Joker seated](https://github.com/vladmandic/automatic/assets/4144634/f45de497-66d3-4380-9b1e-af81e34152ed)

**Reply by weirdlighthouse** [reply]
sorry for late reply but what to do exactly change it to CPU or default

**Reply by xedis** [reply]
> sorry for late reply but what to do exactly change it to CPU or default



@weirdlighthouse 
CPU = More deterministic 
Default = Less Deterministic

**Reply by weirdlighthouse** [reply]
![Capture gdgdgdgd](https://github.com/vladmandic/automatic/assets/140368252/2df19241-3595-4fa5-a5de-841e73c42497)
Am I doing this correct?

**Reply by xedis** [reply, marked as answer]
Just try it, it wont hurt anything if it doesn't work!

**Reply by weirdlighthouse** [reply]
ok

**Comment by StableInfo** [comment]
What were your exact parameters?I would like to try it. Tha,nks

**Reply by xedis** [reply]
@StableInfo did u check the metadata? it should be intact. 
1st image: 
```
Prompt: "a visually stunning composition featuring the Joker seated at a card table, surrounded by an elaborate arrangement of playing cards, chips, and dice, the artwork conveys a sense of calculated madness and the unpredictable nature of his schemes"
Steps: 20 | Seed: 563107175291496 | Sampler: DPM 2M | CFG scale: 6 | Size: 1024x1208 | Parser: Full parser | Model: xl_crystalClearXL_ccxl | Model hash: 0b76532e03 | Refiner: sdXL_v10RefinerVAEFix | VAE: sdxl_vae2 | Image CFG Scale: 6 | Backend: Diffusers | Version: ee2308c | Operations: txt2img | Pipeline: StableDiffusionXLPipeline | CFG rescale: 0.7
```

2nd image : 
```
Prompt: "a visually stunning composition featuring the Joker seated at a card table, surrounded by an elaborate arrangement of playing cards, chips, and dice, the artwork conveys a sense of calculated madness and the unpredictable nature of his schemes"
Steps: 20 | Seed: 2552010018 | Sampler: DPM 2M | CFG scale: 6 | Size: 1024x1208 | Parser: Full parser | Model: xl_icedcoffeexl_v003 | Model hash: a2cf9d3deb | Refiner: sdXL_v10RefinerVAEFix | VAE: sdxl_vae2 | Image CFG Scale: 6 | Backend: Diffusers | Version: ee2308c | Operations: txt2img | Pipeline: StableDiffusionXLPipeline | CFG rescale: 0.7

```
